tests/test_vec_monitor.py

```diff
@@ -45,6 +49,41 @@ def test_vec_monitor(tmp_path):
     os.remove(monitor_file)
 
 
+def test_vec_monitor_info_keywords(tmp_path):
+    """
+    Test loggig `info_keywords` in the `VecMonitor` wrapper
+    """
+    monitor_file = os.path.join(str(tmp_path), f"stable_baselines-test-{uuid.uuid4()}.monitor.csv")
+
+    register(
+        id="BitFlippingEnv-v1",
+        entry_point=BitFlippingEnv,
+    )
+    env = DummyVecEnv([lambda: gym.make("BitFlippingEnv-v1")])
```

Comment:
please use `make_vec_env` and you can pass the `BitFlippingEnv` as argument, you don't need to register the env